data/changelog.txt

```diff
@@ -400,35 +236,6 @@ Add error-checking of recipe (sub)categories.
 Unblacklist most remaining blacklisted mods for automated consistency checking.
 Report log messages for failed tests.
 Reinstate blueprint autocalc.
-Add a compiler error problem matcher.
-Fix bio_batteries failing tests
-Add check enforcing almost-never-auto convention
-Move more player functions into Character
-Use string_id for mutation_category_trait ids
-Add --list/-L argument to keys.py/values.py
-Add documentation for mutation, comestible, and gunmod fields
```

Comment:
It is helpful to know when one should take another look at a previously-read document.